bots/configure-bot-environment/README.md

```diff
@@ -1,57 +1,44 @@
-# Configuring bot environments
+# Configure bot environments
 
-Some configurations are needed by the SDK and are independent of bot type (i.e.
-Hubot, Botkit, Rasa etc).
+Regardless of bot type (e.g. Hubot, Botkit, Rasa, etc.), Rocket.Chat SDK requires you to make some configuration.
```

Comment:
> Rocket.Chat SDK requires you to make some configuration.

the Rocket.Chat SDK requires you to configure some variables for your environment.
or
the Rocket.Chat SDK requires you to configure your environment with some variables